examples/topics/graph/airports_graph.py

```diff
@@ -1,3 +1,15 @@
+'''
+This example creates an interactive map visualization of airport routes and locations using Bokeh.
+
+.. bokeh-example-metadata::
+    :sampledata: airport_routes, us_states
+    :apis: bokeh.models.ColumnDataSource, bokeh.models.StaticLayoutProvider,
+           bokeh.models.NodesAndLinkedEdges, bokeh.models.TapTool, bokeh.plotting.figure,
+           bokeh.plotting.show
```

Comment:
Unfortunately our homegrown sphinx extension is not smart enough to handle multiple lines, yet, so every thing will need to go on a single line. We have a fairly large column limit (160 I think) but you still may want to remove some of these for compactnness